### PR TITLE
Sentiment Analysis functionality added

### PR DESCRIPTION
**Pull request by ayush1999**
The model was trained in Keras and loaded into Flux using [Keras.jl](https://github.com/ayush1999/Keras.jl). The current usage is:

```
>>> using TextAnalysis
>>> model = TextAnalysis.SentimentAnalyser()
>>> model("...")
```

The binaries will be downloaded if not found.

### CODE REVIEW

**Comment by aviks** [comment]
Thanks Ayush, I love this. 

The only thing I'd like to think about is where to store the weights. Should we store it in this repo? A separate repo like the builders? Or a single repo for all Flux pre-trained weights -- maybe the model zoo project itself? 

Also, wherever we store it, I'd like to use the BinaryBuilder infrastructure to download it to the client. 

cc: @MikeInnes for thoughts?

File: src/sentiment.jl [hunk 1]
```diff
@@ -0,0 +1,57 @@
+using Keras
```
Comment (aviks) [inline src/sentiment.jl]: Keras and Flux should probably be added to REQUIRE.

**Comment by MikeInnes** [comment]
RE build, in other projects we've just attached weights to a Github release and downloaded it from there.

@ayush1999 let's get Keras.jl to emit Julia code for the model, like ONNX.jl, and avoid the Keras dependency.

**Comment by ayush1999** [comment]
@aviks Any idea why the build might be failing?

**Comment by aviks** [comment]
The tests are failing only because TextAnalysis master needs Languages master. I can handle that locally, no worries. 

The one change that might be useful is if the hd5 file can be converted to BSON. The we can remove the HDF5 dependency, and add a BSON dependency.  Is that possible?

**Comment by aviks** [comment]
Ayush, thanks for this, I'll take it from here. Really appreciate your effort in getting this done.

**Comment by aviks** [comment]
Closed in favour of #79

**Pull request closed without merging**